gnovm/cmd/gno/repl_test.go

```diff
@@ -11,3 +15,127 @@ func TestReplApp(t *testing.T) {
 	}
 	testMainCaseRun(t, tc)
 }
+
+func TestUpdateIndentLevel(t *testing.T) {
+	tests := []struct {
```

Comment:
It would be good to add some tests that include some quoted indentation-causing characters like `line: "a := '{'` etc.